tests/python-pytest/onnx/export/onnx_backend_test.py

```diff
@@ -80,7 +80,7 @@
     'test_reciprocal',
     'test_sqrt',
     'test_pow',
-    'test_exp',
+    'test_exp_example',
```

Comment:
won't this end up excluding a few tests?

Reply:
'test_exp' includes 'expand' operator tests too which is not implemented yet

Reply:
can you please use `test_exp_'?

Reply:
That will also exclude 'test_exp' test. https://github.com/onnx/onnx/blob/master/docs/Operators.md#Exp

Reply:
ONNX appends `_cpu` to all the node tests. So `test_exp` will be `test_exp_cpu` when it is run. Which is why I think using `test_exp_' might work?